x-pack/test/functional/services/ml/data_visualizer_index_based.ts

```diff
@@ -50,7 +56,10 @@ export function MachineLearningDataVisualizerIndexBasedProvider({
         const filteredCards = await testSubjects.findAll(
           `mlDataVisualizerFieldsPanel ${panelFieldTypes} > ~mlFieldDataCard`
         );
-        expect(filteredCards).to.have.length(expectedCardCount);
+        expect(filteredCards).to.have.length(
+          expectedCardCount,
+          `Expectedfield field card count for panels '${panelFieldTypes}' to be '${expectedCardCount}' (got '${filteredCards.length}')`
```

Comment:
Typo - can just be `Expected field card count...`

Reply:
Fixed in 37f1b9f